ui/src/components/canvas/FlowCanvas.tsx

```diff
@@ -125,6 +124,16 @@ const FlowCanvas = () => {
               <TrashCan />
             </ControlButton>
           </Controls>
+
+          <Controls position="top-left" showFitView={false} showInteractive={false} showZoom={false}> 
+            <ControlButton  title="horizontal layout" onClick={() => updateLayoutOrientation('horizontal')}>
+                <ArrowsHorizontal/>
+              </ControlButton>
+              <ControlButton title="vertical layout" onClick={() => updateLayoutOrientation('vertical')}>
```

Comment:
:question: Can we call `fitView` on the `reactFlowInstance` after updating the layout? So all the nodes appear on-screen after the update. You might have to rely on a `useEffect` in this case, since the node position updates might not be applied until the subsequent render.